src/StdStorage.sol

```diff
@@ -239,6 +239,10 @@ library stdStorage {
         checked_write(self, bytes32(amt));
     }
 
+    function checked_write_signed(StdStorage storage self, int256 val) internal {
```

Comment:
Any objections to this name change to directly reflect the type? This makes it more analogous to how we handle this elsewhere (e.g. `console.logBytes`)

```suggestion
    function checked_write_int(StdStorage storage self, int256 val) internal {
```